File: 3rdPeriod/ObjectOrientedProgramming/Exercicio1/src/Ex6.java
public class Ex6 {
    public static void main(String[] args){
        Double n = 0.0;
        Double menor = Double.POSITIVE_INFINITY, maior = Double.NEGATIVE_INFINITY;
        do{
            try {
                n = EntradaTeclado.leDouble();
            }catch(Exception IOException){
                System.out.println("Insira um número de ponto flutuante!");
            }
            if(n != 0){
                if(n < menor) menor = n;
                if(n > maior) maior = n;
            }
        }while(n != 0);
        System.out.print("O maior é " + maior + " e o menor é " + menor);
    }
}

File: 3rdPeriod/ObjectOrientedProgramming/Exercicio1/ExercicioPython/Ex6.py
class main():
    n = 1.0;
    menor = float('inf')
    maior = float('-inf')
    while(n != 0.0):
        try:
            n = float(input("Insira um número de ponto flutuante: "))    
        except ValueError():
            print("Número inválido! ")
            exit()
        if(n != 0):
            if(n < menor): 
                menor = n
            if(n > maior): 
                maior = n
        
    print("O maior é {} e o menor é {}".format(maior, menor));